frontend/packages/dev-console/src/components/import/GitImport.tsx

```diff
@@ -7,13 +7,15 @@ import { NormalizedBuilderImages, normalizeBuilderImages } from '../../utils/ima
 import { createResources } from './import-submit-utils';
 import { validationSchema } from './import-validation-utils';
 import GitImportForm from './GitImportForm';
+import SourceToImageForm from './SourceToImageForm';
 
 export interface GitImportProps {
   namespace: string;
+  isS2I: boolean;
   imageStreams?: FirehoseList;
 }
 
-const GitImport: React.FC<GitImportProps> = ({ namespace, imageStreams }) => {
+const GitImport: React.FC<GitImportProps> = ({ namespace, imageStreams, isS2I }) => {
```

Comment:
as it's generic now, not sure if it makes sense to give it a generic name rather then `GitImport`

Reply:
Yes, i agree we should rename it to something more generic. But i didn't wanted to rename it in this PR since that will make it very hard to see the actual changes in the logic.

Reply:
Or use `git mv` in a separate commit.